src/View/View.php

```diff
@@ -99,6 +99,7 @@ class View implements EventDispatcherInterface
      * Current passed params. Passed to View from the creating Controller for convenience.
      *
      * @var array
+     * @TODO Deprecate and replace with __get + warning
```

Comment:
Ok, I don't think there is much point in a getter if there is no warning. I'll remove the TODO for now.